examples/config-many-files/teams.yml

```diff
@@ -0,0 +1,27 @@
+# This configuration file allows for the manaing of teams in Fleet.
+# The file can be used for the creating, updating, and deleting of teams.
+# A team's hosts and users are managed with this file.
+
+---
+apiVersion: v1
+kind: teams
+spec:
+  teams:
+  - name: workstations
+    # hosts can only belong to one team
+    hosts:
+      - hardware_serial1
+      - hwardware_serial2
+      - hwardware_serial3
+    # users can belong to multiple teams
+    users:
+      - email1
+  - name: servers
+    hosts:
+      - hardware_serial4
+      - hardware_serial5
+    users:
+      - email: email1
+        role: admin
+      - email: email2
+        role: maintainer
```

Comment:
Are roles pre-defined or user-defined?

If pre-defined, can I see any docs around what the roles mean?
If user-defined, are there config files / specs around how to define roles?

(also I'm guessing that this PR is probably working on conceptualizing several concepts that don't have complete implmentations, so it's a fair response to just say "coming soon")

Reply:
Roles will be pre-defined. 

>  can I see any docs around what the roles mean?

I'm currently working on [a draft of these docs here](https://github.com/fleetdm/fleet/pull/472)! I'll ping you when I make more progress this week.